browser/main/lib/dataApi/createNoteFromUrl.js

```diff
@@ -14,12 +14,18 @@ function validateUrl (str) {
   }
 }
 
+const ERROR_MESSAGES = {
+  ENOTFOUND: 'URL not found. Please check the URL, your internet connection and try again.',
+  VALIDATION_ERROR: 'Please check your URL is in correct format. (Example, https://www.google.com)',
```

Comment:
I would rephrase this as:
`Please check if the URL is in the correct format. (Example, https://www.google.com)`

Reply:
OK. I've changed it.